docker/docker-compose.yml

```diff
@@ -69,6 +69,8 @@ services:
     restart: always
     environment:
       - REDIS_URL=redis://openrelik-redis:6379
+      - OPENRELIK_PYDEBUG=0
+      - OPENRELIK_PYDEBUG_PORT=5678
```

Comment:
@hacktobeer PTAL

Reply:
If we are going to split deployment and development in different docker-compose files the debug settings could be moved to that file. 

Do we really need 2 files? As maintaining 2 different compose files (and adding new worker definitions to them etc) might be more overhead. 

The upside of having this in a production file is that you could enable debugging in a production deployed worker Pod should you want it....

Btw, if you use Tilt (or skaffold) you do not need to change the image name to a local one, it will rewrite the image name automatically and you can use the production docker-compose file for both production and dev.

Reply:
Your right, be able to start debug on a production node could be useful. Let me approve this.